catalog/app/containers/Bucket/requests.js

```diff
@@ -377,8 +376,10 @@ const mergeHits = R.pipe(
 
 export const search = async ({ es, query }) => {
   try {
-    const result = await es(query, {
-      _source: 'key,size,type,updated,user_meta,version_id',
+    const result = await es({
+      query,
+      fields: ['content', 'comment', 'key_text', 'meta_text'],
+      _source: ['key', 'version_id', 'updated', 'size', 'user_meta'],
```

Comment:
i believe that we can remove this line

Reply:
this reduces bandwidth usage significantly by including only specific fields (the ones we need on the front-end) into the response (https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-source-filtering.html)